src/app/technical-record/test-history/test-history.component.ts

```diff
@@ -4,6 +4,7 @@ import { TestResultModel } from '@app/models/test-result.model';
 @Component({
   selector: 'vtm-test-history',
   templateUrl: './test-history.component.html',
+  styleUrls: ['../../app.component.scss'],
```

Comment:
why we loading the app.scss in here?
We shouldn't be having scoped css as agreed.